geometries/Dimer.sdf

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
We should have done this earlier, but we may as well extend the code so it can read `.xyz` files in addition to `.sdf` files so we don't have to keep manually making `.sdf` files.